main_logic/tts_client.py

```
@@ -1498,6 +1502,15 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
 
     try:
         cm = get_config_manager()
+
+        # 检测当前角色是否使用 gsv: 前缀的 voice_id，如果是则强制使用 GPT-SoVITS worker
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)